partials/tabs/coalaonline.html

```diff
@@ -146,7 +146,8 @@ <h4>Optional Settings</h4>
         <div class="row blue-grey darken-4 white-text z-depth-1 no-margin">
           <span>
             <a class="waves-effect grey-text text-lighten-2">
-              <i ng-click="add_bears(section)" class="fa fa-plus-circle bears-icon-small" aria-hidden="true"> &nbsp;ADD BEAR
+              <i ng-click="add_bears(section)" class="fa fa-plus-circle bears-icon-small" aria-hidden="true"> 
```

Comment:
Line contains following spacing inconsistencies:
- Trailing whitespaces.

*Origin: SpaceConsistencyBear, Section: `spacing`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmp01opxl72/partials/tabs/coalaonline.html
+++ b/tmp/tmp01opxl72/partials/tabs/coalaonline.html
@@ -146,7 +146,7 @@
         <div class="row blue-grey darken-4 white-text z-depth-1 no-margin">
           <span>
             <a class="waves-effect grey-text text-lighten-2">
-              <i ng-click="add_bears(section)" class="fa fa-plus-circle bears-icon-small" aria-hidden="true"> 
+              <i ng-click="add_bears(section)" class="fa fa-plus-circle bears-icon-small" aria-hidden="true">
                 &nbsp;<span style="font-family:'Roboto';">ADD BEAR</span>
               </i>
             </a>
```